get_prs_fv3/get_data.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# this scripts downloads test data for the physics standalone
+
+# echo on
+set -x
+
+# get name of standalone package
+cwd=`pwd`
+dirname=`basename ${cwd}`
+
+# remove preexisting data directory
+test -d ./data
+/bin/rm -rf data
+
+# get data
+wget --quiet "ftp://ftp.cscs.ch/in/put/abc/cosmo/fuo/physics_standalone/${dirname}/data.tar.gz"
```

Comment:
Same problem as `get_phi_fv3`

Reply:
I changed the `get_phi_fv3` folder to `phifv3`.